index-singlecell.md

```diff
@@ -76,29 +84,5 @@ Want to talk with trainers and developers on how to improve single cell analysis
 
-- [Pablo Moreno](https://github.com/pcm32)
-- [Mehmet Tekman](https://github.com/mtekman)
-- [Pavankumar Videm](https://github.com/pavanvidem)
-- [Wendi Bacon](https://training.galaxyproject.org/training-material/hall-of-fame/nomadscientist/)
-- [José Domínguez](https://github.com/kysrpex)
-- [Alex Ostrovsky](https://github.com/astrovsky01)
-- [Andrey Solovyev](https://github.com/a-solovyev12)
-- [Bérénice Batut](https://github.com/bebatut)
-- [Bjoern Gruening](https://github.com/bgruening)
-- [Carlos Talavera-Lopez](https://github.com/cartal)
-- [Charles Girardot](https://github.com/cgirardot)
-- [Christophe Antoniewski](https://github.com/drosofff)
-- [Daniel Blankenberg](https://github.com/blankenberg)
-- [Fidel Ramirez](https://github.com/fidelram)
-- [Graham Etherington](https://github.com/ethering)
-- [Helena Rasche](https://training.galaxyproject.org/training-material/hall-of-fame/hexylena/)
-- [Irene Papatheodorou](https://twitter.com/irenepapatheodo)
-- [James Taylor](https://github.com/jxtx)
-- [Jelle Scholtalbers](https://github.com/scholtalbers)
-- [Jonathan Manning](https://github.com/pinin4fjords)
-- [Krzysztof Polanski](https://github.com/ktpolanski)
-- [Lea Bellenger](https://github.com/bellenger-l)
-- [Maria Doyle](https://github.com/mblue9)
-- [Ni Huang](https://github.com/nh3)
-- [Nicola Soranzo](https://github.com/nsoranzo)
-- [Suhaib Mohammed](https://github.com/suhaibMo)
+| <ul><li>[Pablo Moreno](https://github.com/pcm32) </li><li>[Mehmet Tekman](https://github.com/mtekman) </li><li>[Pavankumar Videm](https://github.com/pavanvidem) </li><li>[Wendi Bacon](https://training.galaxyproject.org/training-material/hall-of-fame/nomadscientist/) </li><li>[José Domínguez](https://github.com/kysrpex) </li><li>[Alex Ostrovsky](https://github.com/astrovsky01) </li><li>[Andrey Solovyev](https://github.com/a-solovyev12) </li><li>[Bérénice Batut](https://github.com/bebatut) </li><li>[Bjoern Gruening](https://github.com/bgruening) </li><li>[Carlos Talavera-Lopez](https://github.com/cartal) </li><li>[Charles Girardot](https://github.com/cgirardot) </li><li>[Christophe Antoniewski](https://github.com/drosofff) </li><li>[Daniel Blankenberg](https://github.com/blankenberg) | <ul><li>[Fidel Ramirez](https://github.com/fidelram) </li><li>[Graham Etherington](https://github.com/ethering) </li><li>[Helena Rasche](https://training.galaxyproject.org/training-material/hall-of-fame/hexylena/) </li><li>[Irene Papatheodorou](https://twitter.com/irenepapatheodo) </li><li>[James Taylor](https://github.com/jxtx) </li><li>[Jelle Scholtalbers](https://github.com/scholtalbers) </li><li>[Jonathan Manning](https://github.com/pinin4fjords) </li><li>[Krzysztof Polanski](https://github.com/ktpolanski) </li><li>[Lea Bellenger](https://github.com/bellenger-l) </li><li>[Maria Doyle](https://github.com/mblue9) </li><li>[Ni Huang](https://github.com/nh3) </li><li>[Nicola Soranzo](https://github.com/nsoranzo) </li><li>[Suhaib Mohammed](https://github.com/suhaibMo) |
```

Comment:
why is this change needed?

Reply:
Table looks better than the absurdly giant contributors list